AdminWebsite/AdminWebsite/ClientApp/src/app/services/video-hearings.service.ts

```diff
@@ -117,7 +118,19 @@ export class VideoHearingsService {
 
     saveHearing(newRequest: HearingModel): Promise<HearingDetailsResponse> {
         const hearingRequest = this.mapHearing(newRequest);
-        return this.bhClient.bookNewHearing(hearingRequest).toPromise();
+        const bookingRequest = new BookHearingRequest({
```

Comment:
You don't need this ctor here

Reply:
The ctor allows us to initialise fields that are defined.